docs/WritingTests.asciidoc

```diff
@@ -346,34 +346,87 @@ In short, writing multi-machine tests adds a few more layers of complexity:
 3. actual technical realization (i.e. <<Networking.asciidoc#networking,custom networking>>)
 
 ==== Job dependencies
-There are different dependency *types* (see subsequent section). Additionally, dependencies can be machine-specific (see "Inter-machine dependencies" section).
+There are three dependency *types*: `CHAINED`, `DIRECTLY_CHAINED` and `PARALLEL`
```

Comment:
I think a reader benefits more and might be less confused if on this level we only distinguish between chained and parallel dependencies and also not use the capital case control words but plain english referring to the section headers below.. The directly-chained dependency is anyway explained below the section "Chained dependencies".

Reply:
I was also thinking about that because `CHAINED` and so on are actually "keywords" the user will never have to type literally. Not sure about directly chained. Then I would also go back to "There are different dependency types, i.a. chained and parallel dependencies." (because "chained" and "directly-chained" dependencies are still distinct types).

Reply:
I removed the keywords and mention directly chained dependencies only in the sub section.